desktop/cmp/contextmenu/ContextMenuItem.js

```diff
@@ -11,13 +11,13 @@ import {assign} from 'lodash';
 
 /**
  *  Basic Model for an item displayed within a generic ContextMenu.
- *  @see StoreContextMenuItem for a more specific (and common) implementation tied to data views.
+ *  @see StoreActionDefinition for a more specific (and common) implementation tied to data views.
  */
 export class ContextMenuItem {
 
     text;
     icon;
-    action;
+    actionFn;
     items;
```

Comment:
John and I discusssed this name change, due to the fact that the overall object is likely to be called "action".  Will need to determine if this is worth the risk and makes sense.